products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -1,15 +1,159 @@
-# Global Documentation Search
+# Documentation Search MVP
 
-Author: Bill Fienberg
-Last Updated: 2020-01-23
-Reviewers: TBD
+**Author(s):** Bill Fienberg <bill.fienberg@oddball.io>
 
-## Objective
+**Last Updated:** 2020-02-04
 
-## Background
+**Status:** **Draft**
+
+**Approvers:** Andrew G., Megan K., Rian F., VSP Ops
 
 ## Overview
 
-## Detailed Design
+### Objective
+
+Run our own instance of [Algolia's DocSearch](https://community.algolia.com/docsearch/what-is-docsearch.html) on a custom landing page.
```

Comment:
"run Algolia" sounds like a proposed solution, not the objective of the design doc?

Reply:
I've gone through a couple iterations of the objective. Which one do you think makes the most sense?
- Original: We want to implement a public landing page where VFS/VSP team members can search multiple documentation source from a single entrypoint.
- Generic: Search multiple documentation sources from a single, public entrypoint.
- Specific: Run our own instance of [Algolia's DocSearch](https://community.algolia.com/docsearch/what-is-docsearch.html) on a custom landing page.

Do you consider the Original or Generic option better than the current option (Specific)?

Reply:
I think I'm largely trying to reconcile the stated purpose ("implement Algolia") with the contents (where you talk about needing an indexer and a DB? but those are provided by Algolia, so if the objective is "implement Algolia" then that should be assumed).

Is this doc about "Algolia is already decided on, here's a doc on how we should implement it", or is it "we need a search system, here are some options (and Algolia is a strong contender)"?

Reply:
This doc is about "Algolia is already decided on, here's a doc on how we should implement it."